doc/Advanced-configuration.md

```diff
@@ -107,24 +107,51 @@ The tuple order is important, unless the no `host_config` option is set. Retaini
 
 * **auth_method** (local)
     * **Description:** Chooses an authentication module or a list of modules. Modules from a list are queried one after another until one of them replies positively.
-    * **Valid values:** `internal` (Mnesia), `odbc`, `external`, `anonymous`, `ldap`
-    * **Warning:** `external` and `ldap` limit SASL mechanisms list to `PLAIN` and `ANONYMOUS`.
+    * **Valid values:** `internal` (Mnesia), `odbc`, `external`, `anonymous`, `ldap`, `jwt`, `riak`, `http`
+    * **Warning:** `external`, `jwt` and `ldap` work only with `PLAIN` SASL mechanism.
     * **Examples:** `odbc`, `[internal, anonymous]`
 
-* **auth_password_format** (local)
+* **auth_opts** (local)
+    * **Description:** It it a configuration of chosen auth backends.
```

Comment:
Would be more easy to read:
Description: Auth backend options

Also, matches subsection below.